modules/friendlytag.js

```diff
@@ -1074,9 +1074,10 @@ Twinkle.tag.fileList = {
 					required: true
 				},
 				{
-					type: 'input',
+					type: 'number',
 					name: 'DoNotMoveToCommons_expiry',
 					label: 'Expiration year: ',
+					min: new Morebits.date().getYear() + 1900,
```

Comment:
```suggestion
					min: new Morebits.date().getFullYear(),
```
.getYear() is deprecated.